.github/workflows/main.yml

```diff
@@ -4,10 +4,12 @@ on:
   pull_request:
     branches:
       - newsite
```

Comment:
you can remove `newsite` here and further down

Reply:
Yeah, I was testing with both branch. I will remove it in your PR.